apps/system/utils/kdbg_utils.h

```diff
@@ -38,4 +39,31 @@ struct stkmon_save_s {
 };
 #endif
 
+#if defined(CONFIG_ENABLE_IRQINFO_CMD) || defined(CONFIG_ENABLE_IRQINFO_CMD) || defined(CONFIG_ENABLE_PS_CMD) || defined(CONFIG_ENABLE_STACKMONITOR_CMD)
+
+#define PROC_MOUNTPOINT "/proc"
```

Comment:
Let's tidy up this.
1. If you want to fix a name of proc mount point, let's move this at some public header.
2. If you want to give an option for it, let's make a new config and use it at all of this kind definition.